pokedex_async_redux/lib/state/action/pokemon_actions.dart

```diff
@@ -2,11 +2,16 @@ import 'dart:async';
 
 import 'package:async_redux/async_redux.dart';
 import 'package:pokedex_async_redux/api/api_service.dart';
+import 'package:pokedex_async_redux/state/action/action.dart';
 import 'package:pokedex_async_redux/state/app_state.dart';
 import 'package:pokedex_async_redux/utilities/constants.dart';
 
 /// Setting of pokemons from pokemon api
-class GetPokemonsAction extends ReduxAction<AppState> {
+class GetPokemonsAction extends LoadingAction {
+  static const key = pokemonActionKey;
```

Comment:
this is already a constant right? therefore no need to create a separate one for this 😉